components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -782,6 +782,23 @@ - (NSString *)accessibilityValue {
   return value;
 }
 
+- (NSString *)accessibilityLabel {
+  NSMutableArray *accessibilityStrings = [[NSMutableArray alloc] init];
+  if ([super accessibilityLabel].length > 0) {
```

Comment:
Consider storing `[super accessibilityLabel]` in a local variable so that we're not calling super multiple times here.

Reply:
We are only calling it 2 times. I dont think thats needed.

Reply:
This is not about performance, it's about possible unintended side effects from calling super multiple times. In general we should not be calling super multiple times within a subclass API.

Reply:
Please fix in a follow-up PR.

Reply:
Apologies my statement was less clear about the desired change in the original comment.